src/MongoDB.Driver.Core/Core/Operations/RetryabilityHelper.cs

```diff
@@ -96,24 +100,36 @@ public static void AddRetryableWriteErrorLabelIfRequired(MongoException exceptio
             }
         }
 
+        public static bool IsNetworkException(Exception exception)
```

Comment:
Did existing tests fail without this check?  I don't see any new ones.

Reply:
What is higher than 4.4?

Reply:
>the servers higher than 4.4.

I meant `>= 4.4`, in other words, servers which allow sending error labels